minemeld/flask/sns.py

```diff
@@ -0,0 +1,105 @@
+import requests
+import json
+import os.path
+import uuid
+import minemeld
+from .logger import LOG
+from . import config
+
+SNSAPIURL = config.get('SNS_URL', 'https://o11kiw0vpe.execute-api.us-east-1.amazonaws.com/run')
+SNSENABLED = config.get('SNS_ENABLED', False)
+UUIDFILENAME = config.get('UUID_FILE', 'uu.id4')
+TYPEHELLO = 'hello'
+TYPEMKWISH = 'mkwish'
+TYPESTATS = 'stats'
+
+sns_obj = {}
```

Comment:
could you make both sns_obj and sns_available uppercase as they are globals ?

Reply:
OK